frontend/src/components/professor/GameManagement/GameSettings/textColorHelper.js

```diff
@@ -10,3 +14,9 @@ export const getTextColor = (bgColor) => {
 
   return 'white'
 }
+
+export const isColor = (strColor) => {
+  const s = new Option().style
+  s.color = strColor
+  return s.color !== ''
+}
```

Comment:
To wygląda jak funkcja która jest wykorzystywana tylko do testów - moim zdaniem lepiej przerzucić to do folderu tests/utils(/helpers?)... chociaż nie wiem jak to nazwać wtedy, może po prostu zdefiniować funkcję przed pierwszym describe?